### PR TITLE
Added support for RHEL8.

### PR DESCRIPTION
**Pull request by matejak**
Also refactored the script for better maintainability.

### CODE REVIEW

File: remote_vnc_backend_template.sh [hunk 1]
```diff
@@ -11,33 +11,80 @@ if [ $? -ne 0 ]; then
...
+
+	setup_vncserver_systemd
+
+	post_setup_vncserver_rhel8
 else
 	echo "Error: Unknown distro: '$DISTRO_VERSION'" 1>&2
 	exit 1
 fi
+
+gpasswd -a "<USERNAME> wheel
```
Comment (matusmarhefka) [inline remote_vnc_backend_template.sh]: Missing double quote at the end here.

Comment (matusmarhefka) [reply remote_vnc_backend_template.sh]: Actually these are 2 parameters so it must be either:
`gpasswd -a <USERNAME> wheel`
or
`gpasswd -a "<USERNAME>" "wheel"`

**Comment by matusmarhefka** [comment]
There is a problem with fonts, therefore on RHEL8 we also need to install Workstation group:
`yum groupinstall -y 'Workstation'`

Another issue is with connecting to vnc server, with the default command `vncviewer IP` there is a segfault:
```
...
 TLS:         TLS Handshake failed: An illegal TLS extension was received.

Segmentation fault (core dumped)
```

When I use `vncviewer SecurityTypes=VncAuth IP` problem disappears. I would suggest to rather use this command as default until vnc is fixed.

**Comment by matejak** [comment]
Thank you for your findings, I have updated the PR.

**Comment by matusmarhefka** [comment]
Wonderful :smile:

**Pull request merged**